.travis.yml

```diff
@@ -1,6 +1,9 @@
 sudo: required
 dist: trusty
 
+notifications:
+  email: false
```

Comment:
we just can't edit `.travis.yml` so we can be sure that we have repeatable builds. I can add a check for changing these sensitive files in the build. But then we should decide if we want emails on or off or optional.